.github/workflows/build.yml

```diff
@@ -349,8 +353,12 @@ jobs:
 
       - name: Upload app to TestFlight with Fastlane
         run: |
-          NEWLINE=$'\n'
-          CHANGELOG="$(git show -s --format=%s)$NEWLINE$GITHUB_REPOSITORY/$GITHUB_REF_NAME @ $GITHUB_SHA${NEWLINE}Xcode ${{ env.XCODE_VERSION }}$NEWLINE$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
+          CHANGELOG="$(git show -s --format=%s)
```

Comment:
It will be evaluated as part of the string, chars should be escaped. I believe you can test it with something like:

```
testvar="$(echo test\")
nextline"
echo "$testvar"
```

That may not be the case for github runner vars (`${{ VAR }}`) which are preprocessed.

Reply:
Right, makes sense.